src/style.css

```diff
@@ -1,3 +1,74 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:500&display=swap');
+
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
```

Comment:
Avoid using tag names for styling. Instead, use class selectors to style the 'a' element.